ca/certificate-authority.go

```diff
@@ -47,6 +49,56 @@ var badSignatureAlgorithms = map[x509.SignatureAlgorithm]bool{
 	x509.ECDSAWithSHA1:             true,
 }
 
+// PKCS#9 attribute for requesting certificate extensions
+var oidExtensionRequest = asn1.ObjectIdentifier{1, 2, 840, 113549, 1, 9, 14}
+
+// OID and fixed value for the "must staple" variant of the TLS Feature
+// extension:
+//
+//  Features ::= SEQUENCE OF INTEGER                  [RFC7633]
+//  enum { ... status_request(5) ...} ExtensionType;  [RFC6066]
+//
+// DER Encoding:
+//  30 03 - SEQUENCE (3 octets)
+//  |-- 02 01 - INTEGER (1 octet)
+//  |   |-- 05 - 5
+var oidTLSFeature = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 1, 24}
```

Comment:
I think this is wrong. Per [RFC7633](https://tools.ietf.org/html/rfc7633#page-10):

```
TLS-Feature-Module-2015 {
       iso(1) identified-organization(3) dod(6) internet(1)
       security(5) mechanisms(5) pkix(7) id-mod(0)
       id-mod-tls-feature-2015(86)}
...
       id-pe-tlsfeature OBJECT IDENTIFIER ::= { id-pe 24 }
```

By my reading that final `1` should be a `0`.